GeneratorInterface/Pythia6Interface/plugins/Pythia6Hadronizer.h

```diff
@@ -44,7 +44,7 @@ class JetMatching;
   
   public:
      Pythia6Hadronizer(edm::ParameterSet const& ps);
-     ~Pythia6Hadronizer();
+     virtual ~Pythia6Hadronizer()  = default;
```

Comment:
Since this already had a declaration, that means in some .cc file there already exists the definition of the destructor. Please change this to `virtual ~Pythia6Hadronizer();`